### DISCUSSION TITLE
Zeitversatz im sae

### DISCUSSION THREAD

**Discussion by JoernBr** [camueller/SmartApplianceEnabler#286] [Installation und Betrieb]
Moin,
ich nutze den sae auf einem Raspberry Pi 4, grundsätzlich funktioniert das mit der Steuerung einiger Geräte auch bei mir. 
Was mir jedoch seit Wochen ein Rätsel ist, ist ein Zeitunterschied zwischen dem Sunnyportal und dem sae. 
Die Uhrzeit und Zeitzone sind auf dem Raspi, im Sunnyportal sowie den genutzten Tasmota-Steckdosen  korrekt eingestellt. Auf der Fritzbox, die als Zeitserver eingestellt ist, ebenfalls. Einen Zeitserver zum Synchronisieren habe ich in der Fritzbox eingestellt.

Meine Frage:
Die Laufzeit für eine Teichpumpe ist eingestellt auf 08.00 - 21.00, ausschließlich mit PV-Überschuß, das Sunnyportal schiebt diese auf 08.59 - 20.59.
Ich versteh es nicht. Weiß jemand Rat?
Siehe hier:
![grafik](https://user-images.githubusercontent.com/92262600/175762663-a2430dc3-194e-4da0-a152-2c8d238fdae1.png)
![grafik](https://user-images.githubusercontent.com/92262600/175762675-9e292215-4c63-4a3f-b5ca-76f1e7a5e1a5.png)

**Comment by camueller** [comment]
Das ist ja das gute Recht des Sunny Home Manager, dass er die Laufzeit entsprechend der vorgegebenen Rahmenbedingungen einplant. Es handelt sich hier nicht um eine Zeitschaltuhr, sondern um ein Energiemanagementsystem. Insofern macht Dein zweiter Screenshot keinen Sinn, weil von 8 bis 20 Uhr nur 12 Stunden sind und Du auch 12 Stunden Laufzeit forderst. Die 12 Stunden Laufzeit laut Screnshot sind übrigens nicht limitiert auf Überschuss - dann hättest Du bei Mindest-Laufzeit eine 0 eintragen müssen (siehe [Doku](https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/Schedules_DE.md)).

**Reply by JoernBr** [reply]
OK, die fehlende "Null" bei Mindestlaufzeit verstehe ich, die gibt vor, nur mit PV-Überschuß zu laufen. Dann blinkt die Ampel grün, wenn die Pumpe läuft.
Bei der maximalen Laufzeit ist die Vorgabe "12:00" doch richtig wenn ich 12 Stunden laufen lassen möchte? 
Der sae fordert dort eine Eingabe, wenn ich das Feld leer lasse, wird es rot...?

**Reply by camueller** [reply]
Wenn Du einen Laufzeit von 12 Stunden willst, solltest Du ein Zeitfenster größer als 12 Stunden definieren, z.B. von 6 Uhr bis 21 Uhr.
